jep/0000/README.adoc

```diff
@@ -0,0 +1,243 @@
+= JEP-0000: Continuous Delivery of Jenkins Components
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="1h,1"]
+|===
+| JEP
+| 0000
+
+| Title
+| Continuous Delivery of Jenkins Components
+
+| Sponsor
+| link:https://github.com/jglick[Jesse Glick]
+
+// Use the script `set-jep-status <jep-number> <status>` to update the status.
+| Status
+| Not Submitted :information_source:
+
+| Type
+| Standards
+
+| Created
+| :bulb: Date (YYYY-MM-DD) :bulb:
+
+| BDFL-Delegate
+| TBD
+
+//
+//
+// Uncomment if there is an associated placeholder JIRA issue.
+//| JIRA
+//| :bulb: https://issues.jenkins-ci.org/browse/JENKINS-nnnnn[JENKINS-nnnnn] :bulb:
+//
+//
+// Uncomment if discussion will occur in forum other than jenkinsci-dev@ mailing list.
+//| Discussions-To
+//| :bulb: Link to where discussion and final status announcement will occur :bulb:
+//
+//
+// Uncomment if this JEP depends on one or more other JEPs.
+//| Requires
+//| :bulb: JEP-NUMBER, JEP-NUMBER... :bulb:
+//
+//
+// Uncomment and fill if this JEP is rendered obsolete by a later JEP
+//| Superseded-By
+//| :bulb: JEP-NUMBER :bulb:
+//
+//
+// Uncomment when this JEP status is set to Accepted, Rejected or Withdrawn.
+//| Resolution
+//| :bulb: Link to relevant post in the jenkinsci-dev@ mailing list archives :bulb:
+
+|===
+
+== Abstract
+
+Maintainers of Jenkins component repositories on GitHub, especially plugins, can opt into continuous delivery (CD).
+In this mode, every successful build of the `master` branch by ci.jenkins.io results in a new release of the component.
```

Comment:
I think this release model has obvious benefits in terms of simplicity and ease of use for Jenkins developers, but to me it also seems relatively undesirable for Jenkins administrators and users, especially for plugins using tools like Dependabot. Many changes to plugins are not user-facing, so I think it would be frustrating as a Jenkins administrator to be frequently prompted to update plugins when there are no meaningful or user-visible changes, especially since updating a plugin requires restarting Jenkins.